### PR TITLE
fix #12381: calculated coordinates and variables: internationalize

### LINKED ISSUE #12381: [Global Calculated Waypoints/Variables] Translate all internationalized content, especially texts
For new global calculated waypoints, all texts are currently english-hardcoded. They must be translated e.g. put into `strings.xml`

### PR DESCRIPTION
**Pull request by eddiemuc**
fix #12381: calculated coordinates and variables: internationalize

This PR, if merged, will internationalize all texts introduced with new calculated coordinates and variables tab

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1544,6 +1545,66 @@
...
+    <string name="calccoord_generate_error_nogeopointselected">No Geopoint selected</string>
+
+    <string name="calccoord_migrate_title">Waypoint Migration</string>
+    <string name="calccoord_migrate_migrate">Migrate</string>
+    <string name="calccoord_migrate_migrate_usernote_praefix">Migrated from</string>
+    <string name="calccoord_migrate_cancel">Cancel</string>
+    <string name="calccoord_migrate_dismiss">Dismiss data</string>
+    <string name="calccoord_migrate_dismiss_usernote_praefix">Dismissed</string>
+    <string name="calccoord_migrate_infotext_markdown">Legacy local calculation coordinate data was found in this waypoint:\n\n`%1$s`\n\nIt is suggested to migrate this to global calculation coordinates as follows:\n\n`%2$s`\n\nwith new cache variables:%3$s\n\nPlease choose:\n- **Migrate** to perform migration as suggested\n- **Dismiss Data** to dismiss the found calculation data for good\n- **Cancel** to continue without migration. You will be asked for migration again next time you open the waypoint for editing. Note however that editing the waypoint coordinates could delete local calculation data</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:1575]: Suggestion (but maybe its also wise to cut this string into pieces, which can be handled easier):

A legacy coordinate calculation was found in this waypoint:\n\n`%1$s`\n\nIt You can migrate this to the new cache calculation coordinates as follows:\n\n`%2$s`\n\nwith new cache variables:%3$s\n\nPlease choose:\n- **Migrate** to perform migration as suggested\n- **Dismiss Data** to dismiss the found calculation data for good\n- **Cancel** to continue without migration. You will be asked for migration again next time you open the waypoint for editing. Note however that editing the waypoint coordinates could delete local calculation data.

File: main/res/values/strings.xml [hunk 2]
```diff
@@ -1544,6 +1545,66 @@
...
+
+    <!-- Formula -->
+    <string name="formula_function_sqrt">Square Root</string>
+    <string name="formula_function_sin">Sinus</string>
+    <string name="formula_function_cos">Cosinus</string>
+    <string name="formula_function_tan">Tangens</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:1581]: Tangent

**Review by Lineflyer — APPROVED** [review]
I just reviewed the strings and made some suggestions. Not mandatory though.

File: main/res/values/strings.xml [hunk 3]
```diff
@@ -1544,6 +1545,66 @@
     <string name="from_clipboard">From clipboard</string>
     <string name="copy_to_clipboard">Copy to clipboard</string>
 
+    <!-- Variables -->
+    <string name="variables_titletext">Use this page to note down values and do calculations necessary for this cache. Tap \'Help\' for more information.</string>
```
[the remark this answers is not in the document]
Comment (eddiemuc) [reply main/res/values/strings.xml:1549]: done

File: main/res/values/strings.xml [hunk 4]
```diff
@@ -1544,6 +1545,66 @@
...
+    <!-- Calculated coordinates -->
+    <string name="calccoord_guided_mode">Guided Mode</string>
+    <string name="calccoord_convert_to_plain">Convert to plain Coordinate</string>
+    <string name="calccoord_remove_spaces">Remove Spaces</string>
+
+    <string name="calccoord_generate_title">Generate Waypoints</string>
+    <string name="calccoord_generate_showonmap">Show on Map</string>
+    <string name="calccoord_generate_waypointnameprefix">Generated</string>
+    <string name="calccoord_generate_error_novarwithrange">No valid variable with ranges found</string>
+    <string name="calccoord_generate_error_novalidgeopoints">No combination returned valid geopoints</string>
```
[the remark this answers is not in the document]
Comment (eddiemuc) [reply main/res/values/strings.xml:1566]: done

File: main/res/values/strings.xml [hunk 5]
```diff
@@ -1544,6 +1545,66 @@
...
+    <string name="calccoord_generate_error_nogeopointselected">No Geopoint selected</string>
```
[the remark this answers is not in the document]
Comment (eddiemuc) [reply main/res/values/strings.xml:1567]: done

File: main/res/values/strings.xml (hunk 2 above)
Comment (eddiemuc) [reply main/res/values/strings.xml:1581]: done

**Pull request merged**